include/xwidgets/xholder.hpp

```diff
@@ -126,9 +126,6 @@ namespace xw
         public:
 
             xholder_impl() = default;
-            xholder_impl(xholder_impl&&) = delete;
-            xholder_impl& operator=(const xholder_impl&) = delete;
-            xholder_impl& operator=(xholder_impl&&) = delete;
```

Comment:
Move semantics are defaulted in `xholder_id` that inherit from `xholder_impl`, making this invalid.